app/processing/payment/get-latest-payment.js

```diff
@@ -0,0 +1,21 @@
+const moment = require('moment')
```

Comment:
No integration tests for this folder at all

Reply:
It doesn't integrate with anything, it's all self contained